src/components/homePage/Arrivals.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import "./arrivals.css";
+
+const Arrivals = () => {
+  return (
+    <>
+      <div className="arrivals">
+        <div className="summer-arrivals">
+          <div className="summer-image">
+            <img src="https://res.cloudinary.com/dbfzfqfhl/image/upload/v1647523467/ecom%20item%20images/home%20page%20images/summer_cloths_qifpi2.jpg" alt="" />
```

Comment:
don't forget to put text in 'alt' attribute  because if the image doesn't load then it will show the alternative text.